source/server/listener_manager_impl.h

```diff
@@ -333,56 +333,79 @@ class ListenerImpl : public Network::ListenerConfig,
   SystemTime last_updated_;
 
 private:
-  typedef std::array<Network::FilterChainSharedPtr, 3> SourceTypesArray;
-  typedef std::unordered_map<std::string, SourceTypesArray> ApplicationProtocolsMap;
-  typedef std::unordered_map<std::string, ApplicationProtocolsMap> TransportProtocolsMap;
+  using SourcePortsMap = absl::flat_hash_map<uint16_t, Network::FilterChainSharedPtr>;
```

Comment:
You mean the switch to flat_hash_map? Or using? re: using that's what clang-tidy asks for in new code so I just switched it since I was switching to flash_hash_map also.

Reply:
I was asking about `using`, thanks for the explanation.